src/components/configure/ExportImport.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import { Upload, message, Form, Select, Input, Button } from 'antd';
+import { Icon } from 'antd';
+import { useSelector, useDispatch } from 'react-redux';
+import { setProjectConfig, getProjectConfig, notify } from '../../utils';
+import { useParams } from "react-router-dom";
+import { get, set, increment, decrement } from 'automate-redux';
+import client from "../../client"
+
+
+const YAML = require('yamljs');
+const yaml = require('js-yaml');
+
+const ExportImport = ({ object }) => {
+    const { projectID, selectedDB } = useParams()
+    const [Type, setType] = useState('');
+    const dispatch = useDispatch()
+
+
+    const projects = useSelector(state => state.projects)
+    const projectName = getProjectConfig(projects, projectID, "name")
+
+    const downLoadJson = e => {
+        e.preventDefault();
+
+        const data = new Blob([JSON.stringify(object)], { type: 'content-type/json' });
```

Comment:
The contents of the JSON/YAML file that is downloaded is not indented properly. Make sure the contents of the file are indented properly